send.ps1

```diff
@@ -48,6 +48,7 @@ else {
 }
 
 if ($env:APPVEYOR_PULL_REQUEST_NUMBER) {
+  $COMMIT_SUBJECT="Pull Request #$env:APPVEYOR_PULL_REQUEST_NUMBER (from $env:APPVEYOR_PULL_REQUEST_HEAD_REPO_NAME): $env:APPVEYOR_PULL_REQUEST_TITLE"

```

Comment:
This is too long. How about not including where the PR came from?

Probably something like this:
```suggestion
  $COMMIT_SUBJECT="PR #$env:APPVEYOR_PULL_REQUEST_NUMBER: $env:APPVEYOR_PULL_REQUEST_TITLE"
```